lib/circuitbox/circuit_breaker.rb

```diff
@@ -232,24 +206,16 @@ def log_event_to_process(event)
       end
     end
 
-    # Logs to Memcache.
-    def log_event_to_stat_store(key)
-      if stat_store.read(key, raw: true)
-        stat_store.increment(key)
-      else
-        stat_store.write(key, 1, raw: true)
-      end
-    end
-
     # For returning stale responses when the circuit is open
     def response_key(args)
       Digest::SHA1.hexdigest(storage_key(:cache, args.inspect.to_s))
     end
 
-    def stat_storage_key(event, options = {})
-      storage_key(:stats, align_time_on_minute, event, options)
+    def count_storage_key(event, options = {})
```

Comment:
It makes me a little uneasy to change the name in the same go as removing something very similarly named. Could you maybe split it into 2 so we know the intermediate steps in case one or the other breaks something? I'm thinking of a git bisect run to determine a failure and see myself debugging the wrong thing.